### DISCUSSION TITLE
Volume control stopped working

### DISCUSSION THREAD

**Discussion by SarjyantShrestha** [prasanthrangan/hyprdots#1031] [Q&A] [answered]
![image](https://github.com/prasanthrangan/hyprdots/assets/75552820/480b7769-e4ec-44f7-9fee-fdf72120ccd3)

I use Fn + F2 /F3 to decrease or increase volume. It stopped working since few days ago. I haven't messed with the volumecontrol.sh script either. I commented out the first three because I like to use Fn key. Also those three stopped working as well. Other control like play-pause, next, previous, mic-mute works fine.

**Comment by kRHYME7** [comment]
can you run the script on the terminal?

**Reply by SarjyantShrestha** [reply]
`./volumecontrol.sh -o i`
ERROR: Output device not found...

that's weird. Audio is working fine and I can control volume through pavucontrol

**Reply by kRHYME7** [reply]
I see.  What's your output of  this cmd? 

``` pamixer --get-default-sink ```

**Reply by SarjyantShrestha** [reply]
Default sink:
36 "Echo Cancellation Sink" "Echo-Cancel Sink"

yea, I am using pipewire echo cancel

**Reply by kRHYME7** [reply]
```pamixer --get-default-sink```

can you provdie the full output with triple backticks  `  so that I can parse and test it. ( I wi'll try to wrap up some lines to fix this)

**Reply by SarjyantShrestha** [reply]
```
❯ pamixer --get-default-sink
Default sink:
36 "Echo Cancellation Sink" "Echo-Cancel Sink"
```

This is actually full output

**Reply by kRHYME7** [reply, marked as answer]
@SarjyantShrestha  can you try this 
https://gist.github.com/kRHYME7/7a8e5786dc49e4a617cbc8b95cf898b0

I'll push a PR if this works on your end

**Reply by SarjyantShrestha** [reply]
It worked! Thank you so much for helping out again.

**Reply by kRHYME7** [reply]
Please use the latest revision, I want to see if this version is good to be Pushed on upstream.
